documentation/API_documentation/QoD_Latency_Bandwidth_User_Story.md

```diff
@@ -0,0 +1,10 @@
+# QoD Stable Bandwidth and Latency API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an application developer belonging to an enterprise, I want to request (using my application server/backend service), stable bandwidth/latency for a specified App-Flow from a Communication Service Provider (CSP), so that I can ensure better quality of experience for our end users during congestion.  |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User<br> **Actors:** Application service providers, hyperscalers, application developers. Currently the API does not explicitly handle consent management (expects that the customer either owns the device subscriptions or has agreed to manage consent as a part of general Terms and Conditions) and hence end users are not included as Actors. As this changes, the actor list will be extended with end users. <br> **Scope:** *Order To Activate (OTA)* \- Create/Remove/Get service instance\, Get notification about the instance |
+| ***Pre-conditions*** |The preconditions are listed below:<br><ol><li>The Customer:BusinessManager and Customer:Administrator have been onboarded to the CSP's API platform.&nbsp;&nbsp;</li><li><span class="colour" style="color:var(--vscode-unotes-wysTableText)"><span class="font" style="font-family:var(--vscode-editor-font-family)"><span class="size" style="font-size:1em">The Customer:BusinessManager has successfully subscribed to the QoD product from the product catalog. &nbsp;&nbsp;</li><li><span class="colour" style="color:var(--vscode-unotes-wysTableText)"><span class="font" style="font-family:var(--vscode-editor-font-family)"><span class="size" style="font-size:1em">The Customer:Administrator has onboarded the Customer:User to the platform .</li><li><span class="colour" style="color:var(--vscode-unotes-wysTableText)"><span class="font" style="font-family:var(--vscode-editor-font-family)"><span class="size" style="font-size:1em">The Customer either owns the device subscriptions or has agreed to the terms and conditions of the CSP for managing consent of subscription owners.&nbsp;&nbsp;</li><li><span class="colour" style="color:var(--vscode-unotes-wysTableText)"><span class="font" style="font-family:var(--vscode-editor-font-family)"><span class="size" style="font-size:1em">The access token URL is known to the Customer:User to ensure secure means of API access. |
```

Comment:
"The access token URL is" -> "The access token is"

Reply:
Sentence replacd by "The means to get the access token are known to the Customer:User to ensure secure access of the API." in [1b4169b](https://github.com/camaraproject/QualityOnDemand/pull/19/commits/1b4169b35305c0d488515bf3a9f8f39883cc3841)